app/Http/Controllers/API/WpOrg/Themes/ThemeController.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Http\Controllers\API\WpOrg\Themes;
+
+use App\Http\Controllers\Controller;
+use App\Models\ApiResultsResponse;
+
+class ThemeController extends Controller
+{
+    private $actions = [
+        'query_themes' => 'doQueryThemes',
+    ];
+
+    public function sendResponse($data, $statusCode = 200)
+    {
+        $version = request()->route('version');
+        if ($version == '1.0') {
+            if (is_object($data) && method_exists($data, 'toStdClass')) {
+                $data = $data->toStdClass();
+            }
+            return response(serialize((object) $data), $statusCode);
+        }
+        return response()->json($data, $statusCode);
+    }
+
+    public function info()
```

Comment:
Move this to the top if it's the only public function.

Reply:
properties and constants should stay on top whether private or public.  this ordering is good now.